docker/build.Dockerfile

```diff
@@ -0,0 +1,76 @@
+# backward compatibility:
+FROM ubuntu:12.04.5
```

Comment:
> The older the better

In principle yes but 12.04 will be EOL next month which means that any new vulnerabilities which might crop up in the future won't be patched anymore. Thereby we could end up shipping possibly exploitable code inside the AppImage.